src/components/charts/hubblePlot/Nudge.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Button from '../../site/button';
+import ButtonIcon from '../../site/button/ButtonIcon';
+import ArrowLeft from '../../site/icons/ArrowLeft';
+import ArrowRight from '../../site/icons/ArrowRight';
+import {
+  nudgeContainer,
+  nudgeRight,
+  nudgeLeft,
+  nudgeDown,
+  nudgeUp,
+  nudge,
+} from './hubble-plot.module.scss';
+
+const Nudge = ({
+  show,
+  mouseDownCallback,
+  mouseUpCallback,
+  xValueAccessor,
+  yValueAccessor,
+  xScale,
+  yScale,
+  data,
+  offsetTop,
+  chartScale,
+}) => {
+  const [timer, setTimer] = useState();
+  const [pressing, setPressing] = useState(false);
+  const [direction, setDirection] = useState();
+  const [datum] = data || [];
+  const { [xValueAccessor]: datumX, [yValueAccessor]: datumY } = datum || {};
+
+  const opacity = show ? 1 : 0;
+  const left = `${xScale(datumX) * chartScale}px`;
+  const top = `${(yScale(datumY) + offsetTop) * chartScale}px`;
+
+  useEffect(() => {
+    if (pressing && direction) {
+      mouseDownCallback(direction);
+      const interval = setInterval(() => mouseDownCallback(direction), 100);
```

Comment:
to mimic the action of holding down an arrow key, when the user's mouse is down we fire the nudging callback every 100ms then cancel it when either the mouse is released or moves outside of the button, then also reset the nudging distance.